src/go/lib/config/config_test.go

```diff
@@ -129,14 +129,14 @@ func TestOverrideConfig(t *testing.T) {
 }
 
 func TestDefaultFiles(t *testing.T) {
-	user, _ := user.Current()
+	current, _ := user.Current()
```

Comment:
As for me it can be just `u`

Reply:
I think this is not really important, could be any short name.